src/handlers/displayNameHandler.js

```diff
@@ -23,12 +23,16 @@ export default function displayNameHandler(
   path: NodePath
 ) {
   var displayNamePath = getMemberValuePath(path, 'displayName');
-  if (!displayNamePath) {
-    return;
-  }
-  displayNamePath = resolveToValue(displayNamePath);
-  if (!displayNamePath || !types.Literal.check(displayNamePath.node)) {
-    return;
+
+  if (displayNamePath) {
+    displayNamePath = resolveToValue(displayNamePath);
+    if (!displayNamePath || !types.Literal.check(displayNamePath.node)) {
+      return;
+    }
+    displayNamePath = displayNamePath.node.value
+  } else if (!displayNamePath) {
+    displayNamePath = path.node.id ? path.node.id.name : path.node.declarations[0].id.name;
```

Comment:
Would like a second pair of eyes on this, specifically around ```path.node.declarations[0].id.name```. I'm not sure if I like it.